content/applications/inventory_and_mrp/manufacturing/workflows/continuous_improvement.rst

```diff
@@ -0,0 +1,233 @@
+<../../../services/field_service/onsite_interventions>`. These interventions allow service
+technicians or other employees to be sent to a customer's location to address an issue with a
+product.
+
+Review actions
+==============
+
+Reviewing actions is where the continuous part of continuous improvement comes into play, as it
+allows an organization to evaluate the decisions made in the previous steps. As such, this step is
+essentially returning to the beginning of the process, so that additional problems can be
+identified and addressed.
+
+This means that the Helpdesk and Quality apps should be used again to receive customer and employee
+feedback. Another app that may be useful at this stage is the *Surveys* app.
+
+Surveys
+-------
+
+After implementing changes to a product or process, it may be wise to solicit customers for their
+feedback directly, rather than waiting to hear from them of their own volition. This may bring to
+light feedback that customers may otherwise have neglected to share.
+
+One of the best ways to accomplish this is through the :doc:`Surveys <../../../marketing/surveys>`
+app. Creating a survey and sending it to customers who receive an updated product increases the
+likelihood of receiving relevant feedback about the product.
+
+.. admonition:: Example workflow: coat rack product improvement
+   :class: alert alert-success
+
+   *Wood Hut* is a manufacturer of fine wood products. They are committed to manufacturing products
+   of the highest-possible quality, and are always looking for ways to improve the products they
+   sell, along with the processes used to create them.
+
+   Wood Hut uses the Odoo platform to manage every element of their production, fulfillment, and
+   customer satisfaction processes. They have developed a custom product improvement workflow that
+   incorporates the Helpdesk, Quality, PLM, and Manufacturing apps.
+
+   One of Wood Hut's most popular products is their *coat rack*. It's made entirely of oak, and
+   customers describe it as "sleek and elegant". However, recent customer feedback about the coat
+   rack has brought to light quality issues that necessitate revising the current manufacturing
+   process.
+
+   The product revision workflow begins when the customer service team receives a ticket in the
+   Helpdesk app from a customer having problems with the coat rack she purchased. The customer,
+   Abigail Peterson, has found that her coat rack falls over when more than five coats are hanging
+   from it. This is a major issue, as the coat rack has enough dowels for six coats.
+
+   .. image:: continuous_improvement/helpdesk-ticket.png
+      :align: center
+      :alt: A Helpdesk ticket about an issue with the coat rack product.
+
+   Marc, the customer service employee assigned to the helpdesk ticket, opens the *Quality* app and
+   creates a new quality alert. He tags the *Production Quality Team* and assigns Julie Andreson as
```

Comment:
```suggestion
   creates a new quality alert. He tags the *Production Quality Team*, and assigns Julie Andreson as
```
is it meant to be "Anderson" or the current "Andreson"

Reply:
Andreson